### DISCUSSION TITLE
v0.4.0 - The one with encryption

### DISCUSSION THREAD

**Discussion by justinvdm** [snaplet/docs-old#64] [Changelog]
In this release we are making it possible to encrypt your snapshots. This will allow you to perform encryption during capturing (in cloud or your own machine) and decryption during restoration of snapshots.

- In order to encrypt during share, you’ll need to have a public key in your config (you can opt out using the `--no-encrypt` flag).
- A new option that will generate RSA key-pair in your config directory. Running `snaplet config setup` will store a private key in a `id_rsa` file and a public key in your `config.js`.

**Comment by peterp** [comment]
This feature is 🔥
